### PR TITLE
zh-translation: zh/news/releases/1.11.x/announcing-1.11.4/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
zh/news/releases/1.11.x/announcing-1.11.4/index.md

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.11.x/announcing-1.11.4/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+此版本包含一些漏洞修复，从而提高了系统的稳健性。这个版本说明描述了 Istio 1.11.3 和 Istio 1.11.4 之间的区别。
+
+{{< relnote >}}
+
+## 改变{#changes}
+
+- **修复** 修复了 VMs 能够使用 `--revision` 在 `istioctl x workload entry`
+命令上指定的修订控制平面的问题。
+
+- **修复** 修复了同时创建 Service和 Gateway 导致服务被忽略的问题。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.4/index.md:21]: need a space between Service and `和`

File: content/zh/news/releases/1.11.x/announcing-1.11.4/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - /zh/news/announcing-1.11.4
+---
+
+此版本包含一些漏洞修复，从而提高了系统的稳健性。这个版本说明描述了 Istio 1.11.3 和 Istio 1.11.4 之间的区别。
+
+{{< relnote >}}
+
+## 改变{#changes}
+
+- **修复** 修复了 VMs 能够使用 `--revision` 在 `istioctl x workload entry`
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.4/index.md:18]: Better to translate into:

修复了 VMs 通过在 `istioctl x workload entry` 命令中使用 `--revision` 参数去使用修订版本控制平面的问题

File: content/zh/news/releases/1.11.x/announcing-1.11.4/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+命令上指定的修订控制平面的问题。
+
+- **修复** 修复了同时创建 Service和 Gateway 导致服务被忽略的问题。
+  ([Issue #35172](https://github.com/istio/istio/issues/35172))
+
+- **修复** 修复了一个导致服务入口选择 pod 过时端点的问题。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.4/index.md:24]: Service Entry does not mean `服务入口`

Comment (Xunzhuo) [reply content/zh/news/releases/1.11.x/announcing-1.11.4/index.md:24]: Better to translate into: 
修复了由于 Service Entry 选择 Pods 时导致的 Endpoints 发送更新却无应答的问题

File: content/zh/news/releases/1.11.x/announcing-1.11.4/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## 改变{#changes}
+
+- **修复** 修复了 VMs 通过在 `istioctl x workload entry` 命令中使用 `--revision` 参数
+去使用修订版本控制平面的问题。
+
+- **修复** 修复了同时创建 Service 和 Gateway 导致服务被忽略的问题。
+  ([Issue #35172](https://github.com/istio/istio/issues/35172))
+
+- **修复** 修复了由于 Service Entry 选择 Pods 时导致的 Endpoints 发送更新却无应答的问题。
```
Comment (kebe7jun) [inline content/zh/news/releases/1.11.x/announcing-1.11.4/index.md:24]: 中文文档里面，英文不使用复数。

```suggestion
- **修复** 修复了由于 Service Entry 选择 Pod 时导致的 Endpoint 发送更新却无应答的问题。
```

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**